docs/Contributing.md

```diff
@@ -128,6 +128,38 @@ useful as a comment when reading the code? Or does it only distract?
 Is the explanation independent of the current context and is it likely
 to stand the test of time?
 
+## Write useful tests
+
+New code requires new tests to be written that will exercise that code. This
+applies to bug fixes as well as new features. The tests serve two purposes:
+ensure that the new code does not regress if it needs to be changed and to give 
+an example of what the new code does.
+
+Test cases should be reduced as much as possible.  You can use 
+[Spir-V reduce](https://github.com/KhronosGroup/SPIRV-Tools#reducer) to help
+reduce the size test cases, or do it manually.  This helps other developers 
+understand the tests better, and allows the tests to run faster.
+
+The tests must exist in the LLPC repository. They can be
+
+* a [shaderdb test](../llpc/test/shaderdb), 
+* an [LGC test](../lgc/test),
+* an [LGC unit test](../lgc/unittests), or
+* an [LLPC unit test](../llpc/unittests).
```

Comment:
nit: can you either put llpc unit test after shaderdb. or just above lgc unit test, to keep things in an order